src/components/historyWidget.js

```diff
@@ -141,6 +142,7 @@ export const HistoryWidget = {
       // Set the listening to refresh on widget based on selected reference.
       this.listenRefresh(widget);
     } else {
+      // If there is no references carousel render as a default widget.
```

Comment:
Sugestão: `There is no reference carousel` no singular ou `There are no reference carousels` no plural.